src/editor-mathfield/remote-virtual-keyboard.ts

```diff
@@ -50,7 +50,6 @@ export class VirtualKeyboardDelegate implements VirtualKeyboardInterface {
     this.targetOrigin = options.targetOrigin ?? window.origin;
     this.originValidator = options.originValidator ?? 'same-origin';
     this._mathfield = options.mathfield;
-    this.enable();
```

Comment:
Isn't this call to `enable()` needed here?

Reply:
Good question! No, as it's enabled whenever a `math-field` gets focus. In fact, with this call I found that the keyboard will send commands to the incorrect fields.